Scripts/build.sh

```diff
@@ -68,24 +68,24 @@ function xcbuild() {
     if [ $(which xcpretty) ] ; then
         output_command=xcpretty
     fi
+
```

Comment:
how about the destination variable? do we do anything with it now?

Reply:
Which variable do you mean?

Reply:
I mean I see it is defined, but is not used anywhere

Reply:
It is being used to print output using `xcpretty` if exist, otherwise it will use `cat`. It is used at the end: `xcodebuild "${XC_BUILD_ARGS[@]}" | $**output_command** && exit ${PIPESTATUS[0]}`

BTW I didnt changed anything about that.